heat/core/statistics.py

```diff
@@ -1222,7 +1222,10 @@ def reduce_vars_elementwise(output_shape_i):
 
         if x.lshape[x.split] != 0:
             mu = torch.mean(x._DNDarray__array, dim=axis)
-            var = torch.var(x._DNDarray__array, dim=axis, unbiased=bessel)
+            if x.lshape[x.split] == 1:
+                var = factories.zeros(output_shape_i, dtype=x.dtype, device=x.device)
+            else:
+                var = torch.var(x._DNDarray__array, dim=axis, unbiased=bessel)
```

Comment:
Have you considered that np.var(ddof=1) <=> and np.var <=> torch.tensor(unbiased=False) respectively? In other words, the numpy uses a biased estimator and pytorch an unbiased one by default.
On a single value
- Pytorch's var returns 0 when it's biased.
- numpy's var returns nan when its biased.

Reply:
You're absolutely correct. Thanks a lot. ht.var() and ht.std() are numpy-compliant now